fuse_loss/CMakeLists.txt

```diff
@@ -33,12 +33,18 @@ add_compile_options(-Wall -Werror)
 add_library(${PROJECT_NAME}
   src/arctan_loss.cpp
   src/cauchy_loss.cpp
+  src/dcs_loss.cpp
+  src/fair_loss.cpp
+  src/geman_mcclure_loss.cpp
```

Comment:
Does this mean we need to rebase again?

Reply:
Yes, this is on top of https://github.com/locusrobotics/fuse/pull/142, which is still in review.